FLEDGE.md

```diff
@@ -1168,7 +1208,7 @@ The arguments to this function are:
 
 *   auctionSignals and perBuyerSignals: As in the call to `generateBid()` for the winning interest group.
 *   sellerSignals: The output of `reportResult()` above, giving the seller an opportunity to pass information to the buyer. In the case where the winning buyer won a component auction and then went on to win the top-level auction, this is the output of component auction's seller's `reportResult()` method.
-*   browserSignals: Similar to the argument to `reportResult()` above, though without the seller's desirability score, but with additional `adCost`, `seller`, `madeHighestScoringOtherBid` and potentially `interestGroupName` fields:
+*   browserSignals: Similar to the argument to `reportResult()` above, though without the seller's desirability score, but with additional `adCost`, `seller`, `madeHighestScoringOtherBid` and potentially `buyerReportingId` or `interestGroupName` fields:
```

Comment:
>...and potentially `buyerReportingId` or `interestGroupName` fields

To confirm, subject to k-anon, the buyer can expect 
a) _one_ of the following `browserSignals` attributes: `buyerAndSellerReportingId` OR `buyerReportingId` OR `interestGroupName`
b) `selectedBuyerAndSellerReportingId` may also be provided if it was returned from `generateBid` and is k-anon.

Reply:
>@JensenPaul: Ya, that's probably a good idea.

If you add some copy, I just noticed a typo x2 (fuction -> function) in the descriptions up in §1.2.